app/assets/v2/js/shared.js

```diff
@@ -634,7 +642,7 @@ var retrieveIssueDetails = function() {
     result = sanitizeAPIResults(result);
     if (result['keywords']) {
       var keywords = result['keywords'];
-
+	  
```

Comment:
could we remove this ? ^_^

Reply:
If I remove this eslint throw error: 

`644:7  error  Expected blank line after variable declarations  newline-after-var`

so this empty line in this case is required.